### DISCUSSION TITLE
Using an Amiga formatted compact flash as hard disk

### DISCUSSION THREAD

**Discussion by jman-schief** [BlitterStudio/amiberry#1554] [Q&A] [answered]
I noticed that some options to adding a Hard Disk are not selectable. Before opening a ticket, I wanted to ask if it's something just not implemented. See recording:
https://github.com/user-attachments/assets/d444d7ff-7dfc-4aeb-b5e0-fba7e39edbb9

(also the path to the device is cleared when changing other options)

I wonder what do these four options mean when attaching an hard drive:
- uaehf.device
- IDE (Auto)
- Commodore A600/A1200/A4000
- SCSI (Auto)

might be a useful addition to the [wiki](https://github.com/BlitterStudio/amiberry/wiki/GUI:-HDD).

Thanks!

**Comment by midwan** [comment, marked as answer]
That seems like a bug indeed, it's probably refreshing back to the original value for some reason.
The dropdown selects which Amiga-side device the HDD will be connected to. The first one is internal to the emulator (and the default option), the others are emulated devices of the Amiga hardware.

**Reply by midwan** [reply]
Wiki updated, though the screenshots there need a refresh as well.

**Reply by midwan** [reply]
I've made some slight updates to the code as well, now. The textfield won't be overwritten every time you update a widget, for instance.

**Reply by jman-schief** [reply]
Thank  you :)
